plugins/woocommerce-admin/client/two-column-tasks/task-list.scss

```diff
@@ -0,0 +1,24 @@
+.woocommerce-task-list__item {
```

Comment:
Will this be needed? These new task list designs don't make use of the expanding/collapsing animations that this CSS is for I believe.